compiler/test/unit/frontend.d

```diff
@@ -388,5 +388,6 @@ unittest
     t.module_.fullSemantic();
 
     assert(global.errors == 1);
-    assert(diagnosticMessages[0] == "is a Ddoc file, cannot import it");
+    string msg = "is a Ddoc file, cannot import it";
+    assert(diagnosticMessages[0][$ - 32 .. $] == msg);
```

Comment:
`[$ - msg.length .. $]`

Reply:
I really should stop trying to do something else while I work on this.